packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -47,6 +47,7 @@ function TitleSubHeader({ title, subTitle, onEdit, editable, t }) {
 							theme['tc-subheader-details-text-title-pencil'],
 							'tc-subheader-details-text-title-pencil',
 						)}
+						disabled={disabled}
```

Comment:
You can use `disabled` instead of `disabled={disabled}` for boolean values.

Reply:
no, you can use shorthand only if disabled={true}, there, it is dynamic.